src/js/cf-expandables.js

```diff
@@ -5,110 +5,255 @@
-          duration: duration,
-          easing: 'easeOutExpo'
-        });
-      };
-
-      this.collapse = function( duration ) {
-        $cueOpen.css( 'display', 'inline' );
-        $cueClose.css( 'display', 'none' );
-        $content.attr( 'aria-expanded', 'false' );
-        $target.attr( 'aria-pressed', 'false' );
-        if ( typeof duration === 'undefined' ) {
-          duration = $.fn.expandable.calculateCollapseDuration( $content.height() );
-        }
-        $this.removeClass('expandable__expanded');
-        $content.slideUp({
-          duration: duration,
-          easing: 'easeOutExpo'
-        });
-      };
+      }
 
-      this.init();
+      this.$target.attr( 'aria-controls', this.$content.attr( 'id' ) );
 
-    });
+      if ( $el.hasClass( this.expandedClass ) ) {
+        this.expand( 0 );
+      } else {
+        this.collapse( 0 );
+      }
+    },
 
-  };
+    /**
+     * Initializes the Expandable dom events.
+     */
+    initEvents: function initEvents() {
+      this.$el.on( 'click', '.expandable_target',
+                   _throttle( this.throttleDuration, this.toggle, this )
+                 );
+    },
 
-  $.fn.expandable.calculateExpandDuration = function( height ) {
-    return $.fn.expandable.constrainValue( 450, 900, height * 4 );
-  };
+    /**
+     * Expands or collapses the Expandable.
+     *
+     * @param {object} event - jQuery event object.
+     */
+    toggle: function toggle( event ) {
+      event.preventDefault();
+      event.stopPropagation();
 
-  $.fn.expandable.calculateCollapseDuration = function( height ) {
-    return $.fn.expandable.constrainValue( 350, 900, height * 2 );
-  };
+      if ( this.isInAccordion ) {
+        this.$el.siblings( '.expandable' ).each( function( index, sibling ) {
+          sibling.collapse();
+        } );
+      }
+
+      if ( this.isExpanded ) {
+        this.collapse();
+      } else {
+        this.expand();
+      }
+    },
 
-  $.fn.expandable.constrainValue = function( min, max, duration ) {
-    if ( duration > max ) {
-        return max;
-    } else if ( duration < min ) {
-        return min;
-    } else {
-        return duration;
+    /**
+     * Expands the expandable, sets the internal state
+     * and adds aria attributes.
+     *
+     * @param {integer} duration -
+     * The time duration in which the expansion will occur.
+     */
+    expand: function expand( duration ) {
+      var $content = this.$content;
+      this.$target.attr( 'aria-pressed', 'true' );
+      $content.attr( 'aria-expanded', 'true' );
+
+      if ( typeof duration === 'undefined' ) {
+        duration = $.fn.expandable.calculateExpandDuration( $content.height() );
+      }
+
+      this.$el.addClass( this.expandedClass );
+
+      $content.slideDown( {
+        duration: duration,
+        easing:   'easeOutExpo'
+      } );
+
+      this.isExpanded = true;
+    },
+
+    /**
+     * Collaspses the expandable, sets the internal state
```

Comment:
fixed in https://github.com/sebworks/cf-expandables/commit/4f9e0c59ac1aa93385c859ca507afcd135f72d75